internal/jira/jira_boards.go

```diff
@@ -39,3 +40,16 @@ func (api *httpApi) GetBoardConfiguration(boardId int) (*BoardConfiguration, err
 	}
 	return &result, nil
 }
+
+func (api *httpApi) GetFilter(filterId string) (*Filter, error) {
```

Comment:
would be nice also to add some test for that method  - I'll merge it, and I'll add some test later